api/v1beta1/clusterclass_types.go

```diff
@@ -235,13 +237,15 @@ type WorkersClass struct {
 	// +optional
 	// +listType=map
 	// +listMapKey=class
+	// +kubebuilder:validation:MaxItems=1000
```

Comment:
Well, this sort of defeats the idea of class. might be we should use one of 100,250,500
(same for machine pools)

Reply:
Went with 100